client/src/views/landing.vue

```diff
@@ -105,5 +108,11 @@ export default class Landing extends Vue {
   font-size: 19px;
   font-weight: bold;
   text-transform: uppercase;
+  &:hover {
+    cursor: default;
+  }
+}
+.v-application a {
+  color: white;
```

Comment:
This seems overly broad. Can you specify this more?

Reply:
Absolutely 👍

Reply:
Removed!